python/test/unit/language/test_core.py

```diff
@@ -2312,16 +2312,25 @@ def kernel(X, stride_xm, stride_xk,
     if in_dtype == 'float32' and allow_tf32:
         assert re.search(r'[mma|wgmma.mma_async].sync.aligned.m\d+n\d+k8(?:.row.col)?.f32.tf32.tf32', ptx)
     elif in_dtype == 'float16' and out_dtype == tl.float32:
-        assert re.search(r'[mma|wgmma.mma_async].sync.aligned.m\d+n\d+k16(?:.row.col)?.f32.f16.f16', ptx)
+        if capability[0] == 7 and capability[1] == 5:  # Turing
+          assert re.search(r'[mma|wgmma.mma_async].sync.aligned.m\d+n\d+k8(?:.row.col)?.f32.f16.f16', ptx)
```

Comment:
Why do you search `wgmma` if the architecture is Turing?

Reply:
Fixed